src/resmom/mom_comm.c

```diff
@@ -94,7 +94,9 @@
 #include	"pbs_internal.h"
 #include	"placementsets.h"
 #include	"pbs_reliable.h"
-
+#ifdef PMIX
+#include	"mom_pmix.h"
```

Comment:
extra spaces?

Reply:
Not sure what you're asking for here. Do you want extra space here?

Reply:
i was asking if there are extra spaces here, but it seems there are not.